modules/gapi/test/gapi_gpu_test.cpp

```diff
@@ -0,0 +1,207 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
+
+
+#include "test_precomp.hpp"
+
+
+#include "logger.hpp"
+#include "common/gapi_tests_common.hpp"
+#include "opencv2/gapi/gpu/ggpukernel.hpp"
+#include "opencl_kernels_test_gapi.hpp"
+
+
+namespace cv
+{
+
+#ifdef HAVE_OPENCL
+
+    static void reference_symm7x7_CPU(const cv::Mat& in, const cv::Mat& kernel_coeff, int shift, cv::Mat &out)
```

Comment:
Note: OpenCV's convention is that _output object_ follows all _input objects_, then followed by _parameters_.

So this prototype would look like:
```diff
--- reference_symm7x7_CPU(const cv::Mat& in, const cv::Mat& kernel_coeff, int shift, cv::Mat &out)
+++ reference_symm7x7_CPU(const cv::Mat& in, cv::Mat &out, const cv::Mat& kernel_coeff, int shift)
```

Reply:
It is not a critical issue though, just for a general alignment